### PR TITLE
Wagtail 6.0

### PR DESCRIPTION
**Pull request by katdom13**
[Support Ticket](https://torchbox.monday.com/boards/1124794299/pulses/1396935617)

### Summary of changes

- Added Wagtail 6.0 compatibility
- Added tests for Django 5.0
- Added tests for Python 3.12

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -12,11 +12,11 @@ jobs:
     strategy:
       matrix:
         include:
-          - python: 3.9
-            wagtail: wagtail>=4.1
+          - python: 3.12
```
Comment (rachelhsmith) [inline .github/workflows/test.yaml:15]:
```suggestion
          - python: [3.8, 3.9, 3.10, 3.11, 3.12]
```

Comment (rachelhsmith) [reply .github/workflows/test.yaml:15]: question: I'm wondering if we should add all of the supported python versions to the testing matrix or if that's unnecessary here?

**Review by rachelhsmith — APPROVED** [review]
Hi @katdom13 this LGTM, just a question about testing python versions

File: .github/workflows/test.yaml (hunk 1 above)
Comment (katdom13) [reply .github/workflows/test.yaml:15]: I'm really just following the maintainer's format. Figured they might have a reason not to test every iteration, so I just updated the Python version. I remember a past feedback on one of my PRs regarding that.

Comment (rachelhsmith) [reply .github/workflows/test.yaml:15]: Ah ok. That makes sense, thanks

**Pull request merged**